doc/nrf/releases_and_maturity/migration/migration_sysbuild.rst

```diff
@@ -0,0 +1,688 @@
+
+Update files
+------------
+
+The expected output files are the following:
+
++-----------------------------------------------------+----------------------------------------------------------------------------------------------------------+
+| File                                                | Description                                                                                              |
++=====================================================+==========================================================================================================+
+| ``matter_weather_station/zephyr/zephyr.signed.bin`` | Signed (with MCUboot signing key) main application binary file which can be used directly with MCUmgr    |
++-----------------------------------------------------+----------------------------------------------------------------------------------------------------------+
+| ``signed_by_mcuboot_and_b0_ipc_radio.bin``          | Signed (with b0 and MCUboot signing key) network core IPC radio update from application core binary file |
++-----------------------------------------------------+----------------------------------------------------------------------------------------------------------+
+| ``dfu_multi_image.bin``                             | DFU multi image file containing firmware update files and manifest                                       |
++-----------------------------------------------------+----------------------------------------------------------------------------------------------------------+
+| ``matter.ota``                                      | Matter over-the-air firmware update file                                                                 |
++-----------------------------------------------------+----------------------------------------------------------------------------------------------------------+
+| ``dfu_application.zip``                             | Zip file containing firmware update files and manifest                                                   |
++-----------------------------------------------------+----------------------------------------------------------------------------------------------------------+
+
+.. _child_parent_to_sysbuild_migration_image_overlay_changes:
+
+Image overlay configuration
+===========================
+
+In child/parent image configurations, an application could include additional configuration files in the ``child_image`` folder that would be applied to these images (see :ref:`ug_multi_image_permanent_changes`).
+This feature has been adapted in sysbuild; see :ref:`sysbuild_application_configuration` for an overview.
+You must update child/parent image configuration to use it with sysbuild, as the way these files can be used differs:
+
+* In child/parent image configurations, there can be Kconfig fragments and board overlays that are all used.
+
+* In sysbuild, this is not supported.
```

Comment:
I have a problem parsing the bullet point in question. We state things that aren't supported any more (with sysbuild) but coming paragraphs describe the changes (and adaptations) giving pretty much the same functionality at the end...

So the statement "In sysbuild, this is not supported" is only pointing to ``child_image``fragments and overlays. Maybe the bullet-point can be a bit more clear, so it doesn't seem like we are losing functionality